po/fr.po

```diff
@@ -204,7 +210,7 @@ msgstr "%ls: Aucune clef avec le nom '%ls' n'a été trouvée\n"
 #: src/builtin_bind.cpp:143
 #, fuzzy, c-format
 msgid "%ls: Key with name '%ls' does not have any mapping\n"
-msgstr "%ls: La fonction '%ls' n'existe pas\n"
+msgstr "%ls: La clef avec le nom '%ls' n'a aucune correspondance\n"
```

Comment:
Attention : dans le reste du fichier, « clé » est préféré à « clef » ; autant uniformiser. Par ailleurs, vous pouvez retirer le _fuzzy_.

Beware: in the rest of the file, "clé" is preferred to "clef", so better use the preferred one. In addition, you can drop the _fuzzy_.

Reply:
will do